### PR TITLE
Add section for TN/MPS

### PR DESCRIPTION
**Pull request by trbromley**

### CODE REVIEW

File: doc/releases/changelog-0.37.0.md [hunk 1]
```diff
@@ -6,17 +6,79 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations using the
+  [`quimb` backend](https://quimb.readthedocs.io/en/latest/).
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:12]: This rendered kind of strangely:
image

File: doc/releases/changelog-0.37.0.md [hunk 2]
```diff
@@ -6,17 +6,79 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations using the
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:11]: > performing tensor network and matrix product state simulations

I think this sounds like making tensor networks and contracting them generally, not in the context of circuits.
We could say something more like "... is now available for simulating quantum circuits (faster?) via tensor networks and matrix product states..."

File: doc/releases/changelog-0.37.0.md [hunk 3]
```diff
@@ -6,17 +6,79 @@
...
+  [`quimb` backend](https://quimb.readthedocs.io/en/latest/).
+  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by fixing the
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:18]:
```suggestion
  Either method can be selected when instantiating the `default.tensor` device by setting the
```

File: doc/releases/changelog-0.37.0.md [hunk 4]
```diff
@@ -6,17 +6,79 @@
...
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can allow a large number of qubits to be simulated:
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:20]:
```suggestion
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can can simulate a large number of qubits:
```

File: doc/releases/changelog-0.37.0.md [hunk 5]
```diff
@@ -6,17 +6,79 @@
...
+  With this bond dimension, the expectation values calculated for `default.qubit` and
+  `default.tensor` are different:
+
+  ```pycon
+  >>> np.abs(value_dq - value_mps)
+  tensor(0.0253213, requires_grad=True)
+  ```
+
+  Learn more about `default.tensor` and how to configure it by visiting the
+  [how-to guide](https://pennylane.ai/qml/demos/tutorial_How_to_simulate_quantum_circuits_with_tensor_networks/).
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:81]: Will this link be available soon?

File: doc/releases/changelog-0.37.0.md [hunk 6]
```diff
@@ -6,17 +6,79 @@
...
+      for i in range(num_qubits - 1):
+          qml.CNOT([i, i + 1])
+      return qml.expval(qml.Z(num_qubits - 1))
+
+  circuit()
+  ```
+
+  Additional configuration settings like the matrix product state bond dimension can be specified
+  when instantiating `default.tensor`, see the
+  [device documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:40]: Will this link be available soon?

File: doc/releases/changelog-0.37.0.md [hunk 7]
```diff
@@ -6,17 +6,79 @@
...
+
+  The `default.tensor` device is instantiated with a `max_bond_dim` value:
+
+  ```python
+
+  dev_dq = qml.device("default.qubit")
+  value_dq = qml.QNode(f, dev_dq)()
+
+  dev_mps = qml.device("default.tensor", max_bond_dim=5)
+  value_mps = qml.QNode(f, dev_mps)()
```
Comment (DSGuala) [inline doc/releases/changelog-0.37.0.md:69]: Not the workflow I'm using to seeing with @qml.qnode(dev), but takes up less space :+1: 
Would it make sense to show the improvement we get with higher bond dimension?

**Review by DSGuala — APPROVED** [review]
Looks good to me. Approved with some optional comments/suggestions

File: doc/releases/changelog-0.37.0.md (hunk 2 above)
Comment (trbromley) [reply doc/releases/changelog-0.37.0.md:11]: Thanks, fixed!

File: doc/releases/changelog-0.37.0.md (hunk 1 above)
Comment (trbromley) [reply doc/releases/changelog-0.37.0.md:12]: Thanks, I forgot that - this shows fine in markdown but does not convert correctly to the docs version.

File: doc/releases/changelog-0.37.0.md (hunk 6 above)
Comment (trbromley) [reply doc/releases/changelog-0.37.0.md:40]: Yes, here is the `latest` version:
https://docs.pennylane.ai/en/latest/code/api/pennylane.devices.default_tensor.DefaultTensor.html

File: doc/releases/changelog-0.37.0.md (hunk 5 above)
Comment (trbromley) [reply doc/releases/changelog-0.37.0.md:81]: Yes it should be, but we'll have to remove again if we don't make it :crossed_fingers: 

CC @ikurecic - this is a how-to that we're aiming to release concurrently with the release notes/blog post.

File: doc/releases/changelog-0.37.0.md (hunk 7 above)
Comment (trbromley) [reply doc/releases/changelog-0.37.0.md:69]: I started with that but didn't want to make it too much of a demo - probably best to just keep it simple.

**Review by trbromley — COMMENTED** [review]
Thanks @DSGuala!

**Pull request merged**